python/cudf/cudf/core/series.py

```diff
@@ -3591,6 +3592,81 @@ def second(self):
         """
         return self._get_dt_field("second")
 
+    @property  # type: ignore
+    @_cudf_nvtx_annotate
+    def millisecond(self):
+        """
+        The millisecond of the datetime.
```

Comment:
Match `DateTimeIndex` property and pandas docs by pluralizing:
```suggestion
        The milliseconds of the datetime.
```